### ISSUE TITLE
Running the calculator example creates 4 windows.

### ISSUE THREAD

**Issue by samhattangady** [maps4print/azul#158]
## Description
On running the calculator example with `cargo run --example calculator`, 4 separate windows are created. Closing any of them closes all the windows. The numbers and symbols do not render on calculator buttons. The buttons are blank.

## Version / OS

* Rust version: 
stable-x86_64-unknown-linux-gnu (default)
rustc 1.34.0 (91856ed52 2019-04-10)

* azul version:
commit hash: 9090ce23a114222fcc08dc76b35381e671ab80e7

* Operating system:
Ubuntu 18.04

* Windowing system (X11 or Wayland, Linux only):
X11

## Steps to Reproduce
1. Clone the azul repo at commit hash 9090ce23a114222fcc08dc76b35381e671ab80e7
2. Run `cargo run --example calculator`

**Comment by fschutt** [comment]
Yes, that's intentional (the four windows), that was done in order to test multi-window support. I know about the rendering problems, has to do with webrender not redrawing the full screen. They close all at the same time currently because of problems with handling event loops (right now all windows share an event loop since creating more than one loop leads to segfaults for some reason).
